src/api/general-services/pipeline-manage/constructors/create-new-step.js

```diff
@@ -12,117 +11,25 @@ const createNewStep = (context, step, args) => {
   ) ? 'host.docker.internal'
     : `remoter-server-${experimentId}.${config.pipelineNamespace}.svc.cluster.local`;
 
-  const task = JSON.stringify({
+  const task = {
     experimentId,
     taskName,
     config: processingConfig[taskName] || {},
     server: remoterServer,
-  });
-
-  const stepHash = crypto.randomBytes(5).toString('hex');
-
-  if (config.clusterEnv === 'development') {
-    return {
-      ...step,
-      Type: 'Task',
-      Resource: 'arn:aws:states:::lambda:invoke',
-      // TODO: fix this, add some of old output
-      // see https://docs.aws.amazon.com/step-functions/latest/dg/input-output-resultpath.html
```

Comment:
are we no longer going to use lambdas to run this locally?

Reply:
No, just for the runner itself.

Reply:
that sounds much better -- now we will have the running locally represent better what is actually happening